### ISSUE TITLE
unable to get local issuer certificate

### ISSUE THREAD

**Issue by Dugnist** [nmarus/node-ews#23]
```
let EWS = require('node-ews');

// exchange server connection info
let username = '****@****.**';
let password = '*****';
let host = 'https://mail.****.**';

// initialize node-ews
let ews = new EWS(username, password, host);

let ewsFunction = 'GetUserOofSettings';
let ewsArgs = {
 'Mailbox': {
  'Address':'***@***.**'
 }
};

// query ews, print resulting JSON to console
ews.run(ewsFunction, ewsArgs)
 .then(result => {
  console.log(JSON.stringify(result));
 })
 .catch(err => {
  console.log(err.stack);
 });
```

Catch error: 

```
unable to get local issuer certificate
Error: unable to get local issuer certificate
    at Error (native)
    at TLSSocket.<anonymous> (_tls_wrap.js:1022:38)
    at emitNone (events.js:67:13)
    at TLSSocket.emit (events.js:166:7)
    at TLSSocket._init.ssl.onclienthello.ssl.oncertcb.TLSSocket._finishInit (_tls_wrap.js:586:8)
    at TLSWrap.ssl.onclienthello.ssl.oncertcb.ssl.onnewsession.ssl.onhandshakedone
```

**Comment by nmarus** [comment]
Basic Auth vs NTLM on the server you are connecting to?

**Comment by aaroncalderon** [comment]
I had the same issue, so I put this `process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";` in my script.

**Comment by nmarus** [comment]
Features for Basic Auth is now in version v3.0.0 as well as a notes in README around the 401 error when using NTLM as solved in #26
